torch/csrc/jit/ir.h

```diff
@@ -1011,6 +1011,9 @@ friend struct Block;
     }
     return n;
   }
+  Node* createList(at::ArrayRef<Value*> values) {
```

Comment:
This is dangerous. Someone who doesn't know better is going to call `createList(my_stuff)` in a pass and then fail to ever write a test where there are no elements, leaving a hard to spot bug. At the very least, this should be called `createListFromNonEmptyValues()`

Reply:
It will be a hard assertion if there are no elements, so it's not *that* bad. I can make the name scarier though

Reply:
Turns out it's not used anywhere anymore, so I could remove this!